packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -91,6 +93,18 @@ export const processPatientDocumentRequest = async (
       });
     }
 
+    if (whType === DOWNLOAD_WEBHOOK_TYPE || whType === CONVERSION_WEBHOOK_TYPE) {
+      const progressType = whType === DOWNLOAD_WEBHOOK_TYPE ? "download" : "convert";
+
+      await updateProgressWebhookSent(
```

Comment:
We could condition the call to this function if the flag is not already set, right? This function gets called a lot and this would mean a lot of updates on the patient table (with DB tx and lock for update, which is even more expensive than regular trips to the DB).

Reply:
The logic to prevent whether or not this will be called lives on [this function](https://github.dev/metriport/metriport/blob/401a9ad4d880143fe56ae341c3818845047cdb10/packages/api/src/command/medical/document/process-doc-query-webhook.ts#L1). So if the webhook is sent this will never be triggered